extensions/import/src/wizard/pages/summaryPage.ts

```diff
@@ -0,0 +1,156 @@
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as sqlops from 'sqlops';
+import * as nls from 'vscode-nls';
+import * as sqlstring from 'sqlstring';
+
+import { ImportDataModel } from '../api/models';
+import { ImportPage } from '../api/importPage';
+import { FlatFileProvider, InsertDataResponse } from '../../services/contracts';
+import { FlatFileWizard } from '../flatFileWizard';
+
+const localize = nls.loadMessageBundle();
+
+
+export class SummaryPage extends ImportPage {
+	private table: sqlops.TableComponent;
+	private statusText: sqlops.TextComponent;
+	private loading: sqlops.LoadingComponent;
+	private form: sqlops.FormContainer;
+
+	public constructor(instance: FlatFileWizard, model: ImportDataModel, view: sqlops.ModelView, provider: FlatFileProvider) {
+		super(instance, model, view, provider);
+	}
+
+	async start(): Promise<boolean> {
+		this.table = this.view.modelBuilder.table().component();
+		this.statusText = this.view.modelBuilder.text().component();
+		this.loading = this.view.modelBuilder.loadingComponent().withItem(this.statusText).component();
+
+		this.form = this.view.modelBuilder.formContainer().withFormItems(
+			[
+				{
+					component: this.table,
+					title: localize('flatFileImport.importInformation', 'Import information')
+				},
+				{
+					component: this.loading,
+					title: localize('flatFileImport.importStatus', 'Import status')
+				}
+			]
+		).component();
+
+		await this.view.initializeModel(this.form);
+		return true;
+	}
+
+	async onPageEnter(): Promise<boolean> {
+		this.loading.loading = true;
+		this.populateTable();
+		await this.handleImport();
+		this.loading.loading = false;
+		this.instance.setImportAnotherFileVisibility(true);
+
+		return true;
+	}
+
+	async onPageLeave(): Promise<boolean> {
+		this.instance.setImportAnotherFileVisibility(false);
+
+		return true;
+	}
+
+	private populateTable() {
+		this.table.updateProperties({
+			data: [
+				['Server name', this.model.server.providerName],
+				['Database name', this.model.database],
+				['Table name', this.model.table],
+				['Table schema', this.model.schema],
+				['File to be imported', this.model.filePath]],
+			columns: ['Object type', 'Name'],
+			width: 600,
+			height: 200
+		});
+	}
+
+	private async handleImport(): Promise<boolean> {
+		let changeColumnResults = [];
+		this.model.proseColumns.forEach((val, i, arr) => {
+			let columnChangeParams = {
+				index: i,
+				newName: val.columnName,
+				newDataType: val.dataType,
+				newNullable: val.nullable,
+				newInPrimaryKey: val.primaryKey
+			};
+			changeColumnResults.push(this.provider.sendChangeColumnSettingsRequest(columnChangeParams));
+		});
+
+		let result: InsertDataResponse;
+		let err;
+		try {
+			result = await this.provider.sendInsertDataRequest({
+				connectionString: await this.getConnectionString(),
+				//TODO check what SSMS uses as batch size
```

Comment:
Did we ever figure out what parameter to use here?

Reply:
No, I'll look into it.

Reply:
As discussed this is good to merge for now but we should double check this number